src/modules/folder/folder.repository.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { createMock } from '@golevelup/ts-jest';
+import { CalculateFolderSizeTimeoutException } from './exception/calculate-folder-size-timeout.exception';
+import { SequelizeFolderRepository } from './folder.repository';
+import { FolderModel } from './folder.model';
+
+jest.mock('./folder.model', () => ({
+  FolderModel: {
+    sequelize: {
+      query: jest.fn(() => Promise.resolve([[{ totalsize: 100 }]])),
+    },
+  },
+}));
+
+describe('SequelizeFolderRepository', () => {
+  let repository: SequelizeFolderRepository;
+  let folderModel: typeof FolderModel;
+
+  beforeEach(async () => {
+    folderModel = createMock<typeof FolderModel>();
+
+    repository = new SequelizeFolderRepository(folderModel);
+  });
+
+  describe('calculateFolderSize', () => {
+    const folderUuid = 'your-folder-uuid';
+    const timeoutSeconds = 15;
+
+    it('When calculateFolderSize is called, Then call the query correctly', async () => {
+      const calculateSizeQuery = `
```

Comment:
I do not know if this matching is beneficial or not due to the maintenance implied. What do you think @edisonjpadilla @apsantiso ?

Reply:
By mistake, we could modify the query and since it is of type string it wouldn't give us a syntax error or something like that. The test would be there to validate that the query is being built correctly.

Reply:
Personally, I would not add this kind of test. However, I see not harm in having it. The one things I would remove is the jest.mock above, the folderModel is already Mocked with createMock() isn´t it? If it is necessary then forget what I said. @sg-gs

Reply:
I mean, the maintenance for this is hard, as @edisonjpadilla says, if we change something in the codebase, we do not have anything to alert us. Should we export the string as a constant to use it here and there? Maybe is a good workaround.